libs/pmp-web/repository/data-access/src/lib/+state/repository.actions.ts

```diff
@@ -53,6 +53,8 @@ export namespace fromRepositoryActions {
 
   export class AddRepositorySuccess implements Action {
     readonly type = Types.AddRepositorySuccess;
+
+    constructor(public payload: Repository) {}
```

Comment:
add a dedicated payload interface to be more flexible. For now it will be only {repository; Repository}